<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab3_TemHumiIlluDB</class>
 <widget class="QWidget" name="Tab3_TemHumiIlluDB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDateTimeEdit" name="pStartdatetime_thi"/>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="pEnddatetime_thi"/>
     </item>
     <item>
      <widget class="QPushButton" name="pPBthi">
       <property name="text">
        <string>조회</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pPBclearthi">
       <property name="text">
        <string>초기화</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="pDBthi">
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Temperature</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Illuminance</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Humidity</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
